std/array.d

```diff
@@ -3636,13 +3636,15 @@ if (isDynamicArray!A)
         return app.data;
     }
 
+    import std.format : FormatSpec;
+
     /// ditto
     template toString(Writer)
     if (isOutputRange!(Writer, char))
     {
-        import std.format : FormatSpec;
+        //import std.format : FormatSpec; this doesn't work, see issue 13737
```

Comment:
I need some advise here.
Should I comment out the line or just delete it?

Reply:
Just delete it.